server/views/shared/referralDetails.njk

```diff
@@ -2,11 +2,18 @@
 {% from "govuk/components/tag/macro.njk" import govukTag %}
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
 
 {% set pageTitle = "Referral" %}
 {% set pageSubTitle = "Details" %}
 
 {% block referralPageSection %}
+      {% if presenter.userType == 'probation-practitioner' %}
```

Comment:
same as above here too

Reply:
the general concept is leave the logic to the presenter / view classes, these templates should be kept pretty dumb IMO, just render stuff if it's there.